starters/basic-starter/components/meta.tsx

```diff
@@ -0,0 +1,53 @@
+import Head from 'next/head';
+import { useRouter } from 'next/router';
+import { DrupalMetatag } from 'types/drupal';
+
+interface MetaProps {
+  title?: string;
+  path?: string;
+  tags?: DrupalMetatag[];
+}
+
+export function Meta({ title, tags }: MetaProps) {
+  const router = useRouter();
+
+  return (
+    <Head>
+      <link
+        rel="canonical"
+        href={`${process.env.NEXT_PUBLIC_BASE_URL}${
+          router.asPath !== '/' ? router.asPath : ''
+        }`}
+      />
+      {tags?.length ? (
+        tags.map((tag, index) => {
+          if (tag.attributes.rel === 'canonical') {
+            return null;
+          }
+
+          if (tag.attributes.name === 'title') {
+            return (
+              <title key={tag.attributes.name}>{tag.attributes.content}</title>
+            );
+          }
+          const Tag = tag.tag as keyof JSX.IntrinsicElements;
+          return <Tag key={index} {...tag.attributes}></Tag>;
```

Comment:
It looks like in some cases like `name="image"` and `name="author"` the `content` attribute is an object. We should handle that because otherwise the `content` attribute is rendered as `[object Object]`.

Reply:
yes, agree.

Reply:
@lauriii I've made changes and updated the PR